lib/k8s/stack.rb

```diff
@@ -7,6 +7,10 @@ class Stack
 
     LABEL = 'k8s.kontena.io/stack'
     CHECKSUM_ANNOTATION = 'k8s.kontena.io/stack-checksum'
+    RESOURCE_DEFINITIONS = [
+      'CustomResourceDefinition',
```

Comment:
I don't think `kind` is necessarily unique across the entire API, so strictly speaking these whitelisted resource types should be qualified by the `apiGroup`, as for `PRUNE_IGNORE`...  so these should look something like:

* `apiextensions.k8s.io:CustomResourceDefinition` 
* `apiregistration.k8s.io:APIService`

However, including the complete `apiVersion` (`apiregistration.k8s.io/v1beta1`) would be a bit hazardous, just the apigroup itself should be enough.